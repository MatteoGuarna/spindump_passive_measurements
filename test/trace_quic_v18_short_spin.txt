A short trace between Quant client and server using a draft 18 version of Quant. Not enough packets to get full spin bit measurement in effect.
