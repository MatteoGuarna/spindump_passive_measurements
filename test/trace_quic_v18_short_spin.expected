QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-ba5573c827ffe4e911 (49576:4433) at 1553410393050231 new starting packets 0 0 bytes 0 0
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393101193 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393101193 measurement up right 50962 packets 1 0 bytes 1228 0 bandwidth 1228 0
